### DISCUSSION TITLE
Transform Add field from calculation replace only selected fields.

### DISCUSSION THREAD

**Discussion by pyhas** [grafana/grafana#54370] [Dashboards, Panels & Visualizations]
Hi All,

Is there any way that I can replace only the selected fields when do transform add field from the calculation? The only available option at the moment is to replace all fields with a new field.

![grafana](https://user-images.githubusercontent.com/42034008/187086495-cdc4e608-efa4-4619-b970-f4ff9a9218d3.JPG)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
